src/pages/gutenberg/index.vue

```diff
@@ -149,104 +326,47 @@
       />
       <Label
         align="center"
-        class="text-[20px] text-like-green"
+        :class="['text-[20px]', 'text-like-green']"
         :text="$t('gutenberg_download_subtitle')"
       />
       <div
-        class="items-end justify-center gap-[12px] hidden laptop:gap-[48px] laptop:flex"
+        :class="[
+          'hidden',
+          'laptop:flex',
+
+          'items-end',
+          'justify-center',
+
+          'gap-[12px]',
+          'laptop:gap-[48px]',
+        ]"
       >
-        <div
-          v-for="book of freeDownloadList"
-          :key="book.classId"
-          class="flex flex-col gap-[24px] relative max-w-[220px]"
-        >
-          <img class="shadow-2xl" :src="book.imgSrc" :alt="book.title" />
-          <div
-            class="flex flex-col justify-start gap-[12px] absolute translate-y-[100%] bottom-[-24px]"
-          >
-            <Label align="left" preset="h4" :text="book.title" />
-            <Label
-              align="left"
-              class="text-[14px] text-medium-gray"
-              :text="book.author"
-            />
-            <div class="flex">
-              <GutenbergButton
-                class="w-min mt-[8px]"
-                :text="$t('gutenberg_download')"
-                @click="() => handleClickDownload(book.classId)"
-              />
-            </div>
-          </div>
+        <div v-for="book of freeDownloadList" :key="book.classId">
+          <GutenbergDownloadItem
+            :book-info="book"
+            @download="handleClickDownload"
+          />
         </div>
```

Comment:
Please also update `<GutenbergDownloadItem/>` root compoent to `<li/>` or support `tag="li"` props
```suggestion
        <ul v-for="book of freeDownloadList" :key="book.classId">
          <GutenbergDownloadItem
            :book-info="book"
            @download="handleClickDownload"
          />
        </ul>
```

Reply:
[281b57a](https://github.com/likecoin/liker-land/pull/1468/commits/281b57abd17ba843aaa0e4767a17175896f5d336)